guides/troubleshooting.md

```diff
@@ -47,16 +47,6 @@ To do this, first look at the containerd config file located at
 `/etc/containerd/config.toml`. At the bottom of the file there should
 be a section for each runtime class. For example:
```

Comment:
Thanks for removing the text, but the runtime class output needs to be replaced. If it is just removed this section doesn't make sense.

Reply:
@arvindskumar99 can you address this feedback please. It would be good to get this PR merged for the release since SEV is dropped in other places already